src/lib/builtins.pl

```diff
@@ -33,6 +33,129 @@
 internal settings and basic I/O are all here.
 */
 
+%% (@>)(?X, ?Y)
+%
+% Compares X and Y in the term order mandated by the ISO standard (section 7.2).
+% Terms of different types are sorted in the ascending ordering:
+%
+% 1. Variable
+% 2. Floating point
+% 3. Integer
+% 4. Atom
+% 5. Compound
+%
+% If X and Y are both within one of the five categories, they are ordered as follows.
+%
+% 1. Variables: implementation-dependent, Scryer compares variables according to their
+%    locations in its global and local heap. The address space of the local heap is
+%    always above that of the global heap in the WAM architecture used by Scryer.
+% 2. Floating point and 3. Integer: Ordered by numeric value.
+% 4. X precedes Y if X is the null atom ('') and Y is not, or if X lexicographically
+%    precedes Y.
+% 5. X precedes Y if the arity of X is less than that of Y; if the arities are equal,
+%    X precedes Y if its atom precedes Y's atom according to 4. If the functor names
+%    and arities are equal, the subterms of X and Y are compared recursively.
+
+%% (@<)(?X,Y), (@>=)(?X,?Y), (@=<)(?X,?Y)
```

Comment:
We should put just one predicate per comment. Right now it works because it's very lax in what it accepts but if in the future we want to derive more information it could be problematic. For example, this is going to be problematic for the search box if we want to index these fields too.